src/schema/posts.ts

```diff
@@ -1002,6 +1013,35 @@ export const resolvers: IResolvers<any, Context> = {
         },
       );
     },
+    searchQuestionRecommendations: async (
+      source,
+      _,
+      ctx: Context,
+    ): Promise<GQLPostQuestion[]> => {
+      const res = await ctx.con
```

Comment:
Would graphorm not be preferred here?

Reply:
Makes sense. Let me see how it would be able to easily work with the complex we have here.

Reply:
Done! 🥳